governance/polls/Ratification Poll for Onboard-Amend Keeper Network - Tech-Ops (MIP63c4-SP1) - February 14, 2022.md

```diff
@@ -0,0 +1,59 @@
+---
+title: Ratification Poll for Onboard/Amend Keeper Network - Tech-Ops (MIP63c4-SP1) - February 14, 2022
+summary: This is an onboarding application for the tech-ops internal core unit.
+discussion_link: https://forum.makerdao.com/t/mip63c4-sp1-onboard-amend-keeper-network-tech-ops/12739
+vote_type: Plurality Voting
+categories:
+   - MIPs
+options:
+   0: Abstain
+   1: Yes
+   2: No
+start_date: 2022-02-14T16:00:00
+end_date: 2022-02-28T16:00:00
+---
+# Ratification Poll for Onboard/Amend Keeper Network - Tech-Ops (MIP63c4-SP1) - February 14, 2022
+
+The Governance Facilitators have placed a ratification poll into the [voting system](https://vote.makerdao.com/polling) as part of the responsibilities defined in [MIP51](https://mips.makerdao.com/mips/details/MIP51). This Governance [Poll](https://community-development.makerdao.com/en/learn/governance/on-chain-gov) will be active for fourteen days beginning on Monday, February 14 at 16:00 UTC.
+
+**This is a binary vote.** 
+- **You may vote for a single option.** 
+- **You should vote for the option which you prefer.**
+- **If you would accept either option, you should vote 'Abstain'.**
+
+## Review
+
+The community may vote in this poll to express support or opposition to MIP63c4-SP1 being accepted and implemented in the Maker Protocol.
+
+A brief summary of this proposal has been provided by the MIP Author and is shown below:
+
+*This is an onboarding application for the tech-ops internal core unit. Tech-ops receives a budget external to this proposal, so none is required here. They are already maintaining all keeper operations, so this is a formality of switching to this new standardized Keeper Network system.*
+
+Please review the links below to inform your position on this proposal before voting.
+* [Canonical Proposal Version](https://github.com/makerdao/mips/blob/f746194995010122b41feebe6e5912885d6583c7/MIP63/MIP63c4-Subproposals/MIP63c4-SP1.md)
+* [Latest Proposal Version]($link_to_portal_version)
+* [Proposal Discussion Thread](https://forum.makerdao.com/t/mip63c4-sp1-onboard-amend-keeper-network-tech-ops/12739)
+
+## Outcomes
+
+This poll implements a **Minimum Positive Participation** value. The Minimum Positive Participation is currently set to **10,000 MKR**.
+
+**If the votes for the 'Yes' option exceed the votes for the 'No' option AND the votes for the 'Yes' option exceed 10,000 MKR, then the following actions will be taken:**
+* The MIP Editors will mark the proposal **Accepted** and the Governance Facilitators will confirm its passage on the Governance and Risk call on Thursday, February 24. 
```

Comment:
The date here is wrong, the vote doesn't finish until the 28th, so it would be confirmed on Thursday, March 3rd